infoschema/metric_table_def.go

```diff
@@ -2047,4 +2193,860 @@ var MetricTableMap = map[string]MetricTableDef{
 		PromQL: `node_procs_blocked{$LABEL_CONDITIONS}`,
 		Labels: []string{"instance"},
 	},
+
+	"etcd_wal_fsync_total_count": {
```

Comment:
Should these new table name be added in  executor/inspection_summary.go？

Reply:
No, just for diagnose report.